.gitlab-ci.yml

```diff
@@ -23,13 +23,14 @@ builder-image:
     DOCKER_HOST: "tcp://docker:2375"
     DOCKER_DRIVER: overlay2
     DOCKER_TLS_CERTDIR: ""
+    DOCKER_BUILDKIT: 1
   before_script:
     - echo $CI_JOB_TOKEN | docker login -u gitlab-ci-token --password-stdin $CI_REGISTRY
   script:
-    - cd contrib/containers/ci
```

Comment:
I think there should be two Dockerfiles, `platform` and `platform_i386` and they both should combine with `Dockerfile.finish` to form `ci` while only `platform` and `Dockerfile.finish` is inherited for develop, I think there should be only one canonical CI file to avoid confusion rather than the canonical file being nested in a subdirectory.

Consider using `i386` as the naming scheme instead of `32bit`